unity/Holo/Assets/Editor/ModelPreprocessing/AssetBundleHandling/AssetBundleLoader.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+class AssetBundleLoader
+{
+    private AssetBundle assetBundle;
+    private string bundlePath;
+    private string bundleName;
+
+    private GameObject gameObject;
+    private Mesh mesh;
+
+    [MenuItem("Holo/Load an Asset Bundle")]
+    public static void Load()
+    {
+        AssetBundleLoader assetBundleLoader = new AssetBundleLoader();
+        assetBundleLoader.GetBundleName();
+        assetBundleLoader.LoadBundle();
+        assetBundleLoader.LoadAssetFromBundle();
+    }
+
+    private void GetBundleName()
+    {
+       bundlePath = EditorUtility.OpenFilePanel("Choose AssetBundle to be loaded.", "", "");
+       bundleName = Path.GetFileNameWithoutExtension(bundlePath);
+    }
+
+    private void LoadBundle()

```

Comment:
Definitely this method should have parameter `assetPath` - that way we can reuse this code for collection refresh with external class for gathering all needed AssetBundles